tools/scripts/getWorkerInfo.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
Added description. 

Made `OUTPUT_FILE_ON_DRIVER` to be configurable. 

Note
- `OUTPUT_FILE_ON_WORKER` is intentionally kept constant at `/tmp/system_props.yaml`- since we want the worker to generate the output at this location and fetch it back to the driver at `OUTPUT_FILE_ON_DRIVER`